### PR TITLE
Add branch readme for feature branch as defined by repo process

### PR DESCRIPTION
**Pull request by spbrogan**

### CODE REVIEW

File: docs/BranchReadme.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Branched from: release/202102
+
+Branched on: 12/16/2021
+
+Author: sean.brogan@microsoft.com, tabeebe@microsoft.com
+
+## What's unique about this branch
+
+In this branch a new code first/prototype protocol has been enabled on the QemuQ35 platform.
+Protocol is actually define here in mu_basecore <https://github.com/microsoft/mu_basecore/blob/poc/efi_mem_attributes/MdePkg/Include/Protocol/MemoryAttribute.h>
```
Comment (TaylorBeebe) [inline docs/BranchReadme.md:16]: "defined"

File: docs/BranchReadme.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Unit test here <https://github.com/microsoft/mu_basecore/tree/poc/efi_mem_attributes/MdePkg/Test/ShellTest/MemoryAttributeProtocolFuncTestApp>
+Implemented here <https://github.com/microsoft/mu_basecore/blob/poc/efi_mem_attributes/UefiCpuPkg/CpuDxe/CpuPageTable.c#L1629>
+
+This protocol allows a UEFI compliant loader to change memory attributes on a given memory range.  This will allow
+the DXE Core/UEFI platform to tighten module loading and memory allocations (use NX attribute) while still
+allowing a boot loaders that load additional modules to easily make the code executable.
```
Comment (TaylorBeebe) [inline docs/BranchReadme.md:22]: "allowing boot loaders which..."

**Review by TaylorBeebe — APPROVED** [review]

**Comment by makubacki** [comment]
@spbrogan this is old. I assigned it to you for now.

**Comment by spbrogan** [comment]
branch this is for should now be removed as it has been merged into main

**Pull request closed without merging**